dist/src/main/resources/config/hawkular.properties

```diff
@@ -29,3 +29,4 @@ hawkular-alerts.engine-delay=1000
 hawkular-alerts.engine-period=2000
 hawkular-alerts.base-url=/hawkular/alerts
 #hawkular-alerts.backend-reindex=true
+#hawkular-alerts.prometheus-alerter=true
```

Comment:
Do you think perhaps the alerter should be enabled by default.  I think at the moment the elastic alerter is enabled by default but the p8s alerter is not.  Maybe that is confusing.